stl/inc/xutility

```diff
@@ -3497,7 +3497,7 @@ namespace ranges {
     }
 
     template <size_t _Idx, class _It, class _Se, subrange_kind _Ki>
-        requires (_Idx < 2)
+        requires ((_Idx  == 0 && copyable<_It>) || _Idx == 1)
     _NODISCARD constexpr auto get(subrange<_It, _Se, _Ki>&& _Val) {

```

Comment:
`get(subrange&&)` should not be changed according to LWG-3589.

Reply:
There's "I'm in a hurry trying to implement all the Ranges LWG issues" and then there's "I saw two patches of green so I changed two things without concern for what two things they happened to be".